_data/stage3.yml

```diff
@@ -219,3 +242,35 @@
     */
   resources:
     - https://tc39.github.io/proposal-string-matchall/
+  presented:
+    - date: September 2018
+      url: https://github.com/tc39/tc39-notes/blob/master/es9/2018-09/sept-25.md#update-on-stringprototypematchall
+
+- title: Hashbang Grammar
+  description: This proposal is to match de-facto usage in some CLI JS hosts that allow for Shebangs / Hashbang. Such hosts strip the hashbang in order to generate valid JS source texts before passing to JS engines currently. This would unify and standardize how that is done..
+  authors:
+    - Bradley Farias
+  champions:
+    - Bradley Farias
+  tests:
+    - https://github.com/tc39/test262/pull/2065
+  specification: https://tc39.github.io/proposal-hashbang/out.html
+  resources:
+    - https://github.com/tc39/proposal-hashbang
+  example: >
+    #!/usr/bin/env node
+
+    // in the Script Goal
+
+    'use strict';
+
+    console.log(1);
+
+
+    #!/usr/bin/env node
```

Comment:
We should indicate that this is a first line of another script

Reply:
thanks, opened a bug to have more than one script.